src/api/authorize.ts

```diff
@@ -61,6 +62,7 @@ export default function authorize(storedOptions: StoredOptions, options: Authori
     state,
     country: 'JP',
     locale,
+    saml_subject_id: samlSubjectId,
```

Comment:
can i assumed it will not end up `url?saml_subject_id=undefined` in the case of `samlSubjectId` is `undefined`?

Reply:
Yes, We added a test to assert that also